.codeclimate.yml

```diff
@@ -81,3 +81,4 @@ exclude_patterns:
   - "**/pytest_plugin/"
   - "**/sandbox/"
   - services/web/server/src/simcore_service_webserver/exporter/formatters/sds/xlsx/templates/code_description.py
+  - services/web/server/src/simcore_service_webserver/projects/projects_handlers.py
```

Comment:
why is that?

Reply:
Adding files to this list should be justified with a comment except for obvious cases

Reply:
Ahh yes, I can't seem to make code climate ignore line number checks on that file from the UI. It was done previously by someone but in this PR it no longer works.

Reply:
Please add a case to split ``projects_handlers.py`` or a FIXME note on top of ``projects_handlers.py`` ... we should not just ignore it.
The same applies for line 83.

Reply:
No longer needed and removed.